Essentials/src/com/earth2me/essentials/commands/Commandjump.java

```diff
@@ -46,7 +47,12 @@ public void run(final Server server, final User user, final String commandLabel,
 
         final Trade charge = new Trade(this.getName(), ess);
         charge.isAffordableFor(user);
-        user.getTeleport().teleport(loc, charge, TeleportCause.COMMAND);
+        // If the user is allowed to jump outside of the vanilla world boundary
+        if(Bukkit.getWorld(user.getWorld().getUID()).getWorldBorder().isInside(loc) || user.isAuthorized("essentials.jump.pastWorldBorder")) {
```

Comment:
You should add a space between the "if" and "(" to follow the convention of the other code in this file. Also, likewise, conventionally permissions are lower case and short / to the point. Something like "essentials.jump.border" might be better.